101-batch-with-keyvault-pfx-password/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "type": "QuickStart",
+    "itemDisplayName": "batch account with keyvault pfx password",
```

Comment:
use title case

Reply:
Done. Changed to  "itemDisplayName": "Existing Azure Batch Service with Pfx from Keyvault",